libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.h

```diff
@@ -131,6 +131,9 @@ class AP_AdvancedFailsafe
     AP_Int8  _rc_term_manual_only;
     AP_Int8  _enable_dual_loss;
     AP_Int16  _max_range_km;
+    AP_Int8  _term_pos_ail;
```

Comment:
the other terminate variables start with ``_terminate_`` so please follow that pattern.